tensorboard/plugins/histogram/summary_test.py

```diff
@@ -381,11 +318,19 @@ def test_singleton_input(self):
         np.testing.assert_allclose(buckets, expected_buckets)
 
     def test_input_with_all_same_values(self):
-        pb = self.histogram("twelven", [12, 12, 12], buckets=2)
+        pb = self.histogram("twelven", [12, 12, 12])
         buckets = tensor_util.make_ndarray(pb.value[0].tensor)
-        expected_buckets = np.array([[12, 12, 0], [12, 12, 3]])
+        # By default there will be 30 buckets.
+        expected_buckets = np.array(
+            [[12, 12, 0] for _ in range(29)] + [[12, 12, 3]]
+        )
         np.testing.assert_allclose(buckets, expected_buckets)
 
+    def test_zero_bucket_count(self):
+        pb = self.histogram("zero_bucket_count", [1, 1, 1], buckets=0)
+        buckets = tensor_util.make_ndarray(pb.value[0].tensor)
+        np.testing.assert_allclose(buckets, np.array([]).reshape((0, 3)))
```

Comment:
assert_array_equal makes a bit more sense here IMO than assert_allclose, since there are no actual values to compare, so this is really just a shape comparison.  In general, I'd try to use stricter assertions except in cases where we really need to allow for floating point error.

Reply:
Done.